docs/rules/brace-style.md

```diff
@@ -40,27 +40,27 @@ While no style is considered better than the other, most developers agree that h
 
 ## Rule Details
 
-This rule is aimed at enforcing a particular brace style in JavaScript. As such, it warns whenever it sees a statement or declaration that does not adhere to the one true brace style.
+This rule is aimed at enforcing a particular brace style in JavaScript.
```

Comment:
Yes, “aims to” instead of “is aimed at”

Yesterday, I was thinking that as the rule descriptions become clearer, and when the headings in the rule docs become consistent, this sentence might be redundant. But the repetition might be good: many people seem to scan headings, but not read them expecting to find useful information.